src/main/java/com/cairn/waypoint/dashboard/entity/HomeworkQuestionLinkedProtocolTemplate.java

```diff
@@ -0,0 +1,40 @@
+package com.cairn.waypoint.dashboard.entity;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.hibernate.annotations.SQLRestriction;
+
+@Data
+@Entity
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "protocol_template_linked_homework_question")
+public class HomeworkQuestionLinkedProtocolTemplate {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
```

Comment:
Does this table have an `id` field? If so, let's remove it. There is no need for it and this becomes bloatware. We can then remove this field. Note that in your liquibase you should ADD a new yaml file to remove the column and then remove the field from this class. That should resolve this.